tests/Transport/BaseTestCase.php

```diff
@@ -17,15 +18,17 @@ abstract class BaseTestCase extends TestCase {
 	protected $skip_https = false;
 
 	public function set_up() {
-		$callback  = array($this->transport, 'test');
-		$supported = call_user_func($callback);
+		// Intermediary variable $test_method can be skipped with PHP 7+.
```

Comment:
```suggestion
		// Intermediary variable $test_method. This can be simplified (removed) once the minimum supported PHP version is 7.0 or higher.
```